.gitignore

```diff
@@ -30,3 +30,8 @@ npm-debug.log
 Tests/app/cache
```

Comment:
Tests/app folders should not longer be needed also:

.sass-cache
bower_components